docs/pages/steps/jira_add_comment.md

```diff
@@ -10,10 +10,7 @@ folder: steps
 
```

Comment:
Probably explaining more about input and output is so much helpful. 

* for all the get steps input fields are very simple, but for output we need to mention that it is whatever JIRA returns (with 1.2.0) and also we can mention about https://jenkinsci.github.io/jira-steps-plugin/config.html#common-response--error-handling in each step.
* for rest of the steps where we need more input, that also requires some explanation and probably we should say that users must refer to the rest api documentation appropriately.